ntoskrnl/ke/i386/kiinit.c

```diff
@@ -480,6 +480,12 @@ KiInitializeKernel(IN PKPROCESS InitProcess,
     /* Save CPU state */
     KiSaveProcessorControlState(&Prcb->ProcessorState);
 
+#if DBG
+    /* Print applied kernel features/policies and boot CPU features */
```

Comment:
It would be interesting to show the info also for each other CPU present in the system (since in SMP we can have CPUs with slightly different features, but not too different or incompatible though), whenever those are attempted to be started.

Reply:
Indeed it would be interesting feature, hope we'll get SMP working one day. 🙂